### PR TITLE
fix(UserAvatar): updated sizes

### PR DESCRIPTION
**Pull request by Ratheeshrajan**
Contributes to #2233

Add new size for 48px to User Profile Image

#### What did you change?

Introduced new sizes `xl`, `lg`, `md`, `sm` respective to `64px`, `48px`, `32px`, `24px`.

Carbon [IconButton](https://react.carbondesignsystem.com/?path=/story/components-iconbutton--default) component only accepts [sizing](https://github.com/carbon-design-system/carbon/blob/f7b7b4d9791e4dd5339faf1d2ebb53cfe4b574ff/packages/react/src/components/IconButton/index.js#L126) `size: PropTypes.oneOf(['sm', 'md', 'lg'])`, then the alternative is `Button` variant [Icon Button](https://react.carbondesignsystem.com/?path=/story/components-button--icon-button&args=size:xl). The `Button` component accepts size variants "sm", "md", "lg", `xl`, `2xl` when I tried this variant, this was looking good in UI and Storybook but, console returned warning and test case failed due to "Failed prop type: Invalid prop "size" of value "xl" supplied to ForwardRef(IconButton), expected one of ["sm","md","lg"]". because the button component `hasIconOnly` is also using the [IconButton](https://github.com/carbon-design-system/carbon/blob/f7b7b4d9791e4dd5339faf1d2ebb53cfe4b574ff/packages/react/src/components/Button/Button.tsx#L261) component to render this varient. If we use sizes other than `"sm","md","lg"` will show a prop [validation error](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/IconButton/index.js#L126). So I have just used the `<Tooltip>` and `<Button>` kind `ghost` is used to render User avatar with tooltip. Hope this works.

#### How did you test and verify your work?
Storybook

### CODE REVIEW

**Comment by elycheea** [comment]
@Ratheeshrajan I think we were originally thinking to create a new `UserAvatar` component so we can deprecate the `UserProfileImage`. The `UserProfileImage` would have breaking changes to change the sizes directly.

Might also be a good opportunity to revisit how we handle the theming and backgroundColor since they can sometimes be confusing — you need to modify both the `backgroundColor` and `theme` prop currently but think we have an opportunity to simplify that in the new version.

**Comment by elycheea** [comment]
Was chatting with @Laura-Marshall as well and it sounds like they ran into some similar issues on how to manage the color names in Figma. While she wasn’t one of the original designers on `UserProfileImage`, I think she would be a good reference since Marion is on leave and Mitchell is no longer at IBM.

File: packages/ibm-products/src/components/UserProfileImage/UserProfileImage.js [hunk 1]
```diff
@@ -139,17 +137,31 @@ export let UserProfileImage = React.forwardRef(
...
     return (
       FillItem &&
       (tooltipText ? (
-        <IconButton
-          label={tooltipText}
-          className={`${blockClass}__tooltip`}
-          kind="ghost"
+        <Tooltip
           align={tooltipAlignment}
+          className={cx(
```
[the remark this answers is not in the document]
Comment (Ratheeshrajan) [reply packages/ibm-products/src/components/UserProfileImage/UserProfileImage.js]: @davidmenendez removed `cx'

**Comment by Ratheeshrajan** [comment]
[@elycheea and @davidmenendez I'll look into the opportunities for the new `User avatar` component. Meantime I have some confusion about theming while looking at the [guidelines](url) ](https://pages.github.ibm.com/cdai-design/pal/patterns/user-profile-images/usage#colors), there is two sets of colors `dark` and `light` defined. Is there any way we can switch the background color, eg: if the theme is `white` or `g10` then use the `light` color set?

**Comment by elycheea** [comment]
> @elycheea and @davidmenendez I'll look into the opportunities for the new `User avatar` component. Meantime I have some confusion about theming while looking at the [guidelines](https://pages.github.ibm.com/cdai-design/pal/patterns/user-profile-images/usage#colors), there is two sets of colors `dark` and `light` defined. Is there any way we can switch the background color, eg: if the theme is `white` or `g10` then use the `light` color set?

The guidance is out of date — neither the `UserProfileImage` nor the guidance page is correct so we documented the intended behavior in the new issue. I think it would be good to align on what we want out of the `UserAvatar`.

Do we need to control light and dark separately or can we just use the themes? This and the color naming could be good to bring up in [a discussion](https://github.com/carbon-design-system/ibm-products/discussions). I was chatting with @Laura-Marshall and it sounded like Cate W also saw this issue when the team was building the Figma version of the images. They followed our current approach but if we’re creating a new component, it could be good to see how our consumers expect it to work too.

**Comment by lee-chase** [comment]
WRT backround and foreground color @elycheea what we need is a dark and light version.

**Comment by elycheea** [comment]
@Ratheeshrajan I’m going to close this PR for now. I do think we want to approach this as a newer component rather than updating the current one, but it may help for us to have a smaller kickoff about this with maybe @Laura-Marshall, Tomer, or Cate since the original designers are not here to help with this discussion.

We can check in via Slack since I know you’ve also ramped up a bit in product — we can hopefully set some more realistic timing if you’d still like to work on this in addition to product work.

**Pull request closed without merging**